diff --git a/CMakeLists.txt b/CMakeLists.txt
index a484c2c..680fb7c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,6 +13,35 @@ project(${PLUGIN_NAME} VERSION 1.0.0)
 
 add_subdirectory(JUCE)
 
+# begin Android specifics -->
+# It is not beautiful, but there does not seem to be any way to add custom include directory for modules.
+include_directories(
+        "${AAP_DIR}/include"
+        )
+juce_add_module(${AAP_JUCE_DIR}/aap-modules/juceaap_audio_plugin_client)
+
+if (ANDROID)
+
+# dependencies
+find_library(log "log")
+find_library(android "android")
+find_library(glesv2 "GLESv2")
+find_library(egl "EGL")
+set(cpufeatures_lib "cpufeatures")
+set(oboe_lib "oboe")
+
+add_library("cpufeatures" STATIC "${ANDROID_NDK}/sources/android/cpufeatures/cpu-features.c")
+
+
+include_directories(#${PLUGIN_NAME} PRIVATE
+        "${OBOE_DIR}/include"
+        )
+
+add_compile_definitions([[JUCE_DONT_AUTO_OPEN_MIDI_DEVICES_ON_MOBILE=1]])
+
+endif (ANDROID)
+# <-- end Android specifics
+
 juce_add_plugin(${PLUGIN_NAME}
     COMPANY_NAME "witte"
     IS_SYNTH FALSE
@@ -23,7 +52,7 @@ juce_add_plugin(${PLUGIN_NAME}
     COPY_PLUGIN_AFTER_BUILD FALSE
     PLUGIN_MANUFACTURER_CODE Witt
     PLUGIN_CODE Eq00
-    FORMATS AU VST3
+    FORMATS AU Standalone
     PRODUCT_NAME "Eq")
 
 target_sources(${PLUGIN_NAME} PRIVATE
@@ -54,7 +83,7 @@ juce_add_binary_data(${PLUGIN_DATA} SOURCES
 set_target_properties(${PLUGIN_NAME} ${PLUGIN_DATA} PROPERTIES POSITION_INDEPENDENT_CODE ON)
 
 if (UNIX)
-    target_compile_options(${PLUGIN_NAME} PUBLIC -Wall -Wextra -pedantic -Werror)
+    target_compile_options(${PLUGIN_NAME} PUBLIC -Wall)
 else()
     add_definitions("/wd4244") # disable "possible loss of data" warning/error
     target_compile_options(${PLUGIN_NAME} PUBLIC /W4 /WX)
@@ -66,4 +95,12 @@ target_link_libraries(${PLUGIN_NAME} PRIVATE
     juce::juce_audio_utils
     juce::juce_audio_devices
     juce::juce_gui_basics
-    juce::juce_dsp)
+    juce::juce_dsp
+    juceaap_audio_plugin_client
+        ${log}
+        ${android}
+        ${glesv2}
+        ${egl}
+        ${cpufeatures_lib}
+        #${oboe_lib}
+        )
diff --git a/JUCE b/JUCE
--- a/JUCE
+++ b/JUCE
@@ -1 +1 @@
-Subproject commit 1fc0de32c8154a86030071b61e7a659713ddc099
+Subproject commit 1fc0de32c8154a86030071b61e7a659713ddc099-dirty
